platform/cavium/cavm-sai.mk

```diff
@@ -1,12 +1,20 @@
 # Cavium SAI
 
+CAVM_SAI_GITHUB = https://github.com/garyachy/OpenXPS/raw/sai_0.9.5/SAI/cavm-sai/
```

Comment:
Cavium provides both SAI headers and SAI adapter library.

In case if SONIC is upgraded to use SAI 1.0 headers, Cavium build will be broken until we change both our SAI headers and SAI adapter library.

Anyway, thanks for your review comments.